src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java

```diff
@@ -122,14 +123,10 @@ else if (!subUrl.startsWith("http://")) {
                     }
                     try {
                         logger.info("Retrieving full-size image location from " + parentHref);
-                        Thread.sleep(1000);
```

Comment:
Ah, I didn't realize that was existing behavior. I thought it was introduced in your change. Thanks for fixing it!